src/components/PeopleTable.tsx

```diff
@@ -9,44 +50,68 @@ export const PeopleTable = () => {
           <th>
```

Comment:
The sorting icon logic is incorrect. When 'order' is not defined, 'fa-sort' should be used. 'fa-sort-up' should be used when 'order' is 'asc', and 'fa-sort-down' when 'order' is 'desc'.